src/core_ocean/driver/mpas_ocn_core_interface.F

```diff
@@ -126,6 +126,8 @@ function ocn_setup_packages(configPool, packagePool, iocontext) result(ierr)!{{{
       logical, pointer :: tidalForcingActive
       logical, pointer :: tidalPotentialForcingPKGActive
       logical, pointer :: vegetationDragPKGActive
+      logical, pointer :: sedimentTransportPKGActive
+      logical, pointer :: sedimentTracersPKGActive
```

Comment:
@caozd999 Does this package still exist?

Reply:
The two packages both exist. `sedimentTransportPKGActive` is for general sediment transport, under which we define the sediment transport related variables such as sediment characteristics; `sedimentTracersPKGActive` is specifically for process-based suspended sediment transport.